contracts/TellorFlex.sol

```diff
@@ -580,60 +614,110 @@ contract TellorFlex {
     // *                                                                           *
     // *****************************************************************************
 
+    function _updateStakeAmount() internal {
+        bytes memory val = retrieveData(
```

Comment:
using retrieve data isn't going to work unless there happens to be a trb price submission at exactly `block.timestamp - 12 hours`. we would need to use something akin to `getDataBefore` from usingtellor